packages/cookies-tooltip/cookies-tooltip.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, useEffect, useState } from 'react'
+import { Cookie, CookieInverse } from '@lidofinance/icons'
+import { ContentTheme } from '@lidofinance/content-theme'
+import { getCrossDomainCookieClientSide } from '@lidofinance/utils'
+
+import {
+  Wrap,
+  Box,
+  CookieIconWrap,
+  Text,
+  ButtonsWrap,
+  AllowButton,
+  DeclineButton,
+  Link,
+} from './styles'
+import { allowCookies, declineCookies, setWindowOnfocus } from './utils'
+import { COOKIE_ALLOWED_KEY } from './constants'
+
+export const CookiesTooltip: FC = () => {
+  const [isVisible, setVisibility] = useState(false)
+
+  useEffect(() => {
+    // Check cookie after page loaded
+    if (getCrossDomainCookieClientSide(COOKIE_ALLOWED_KEY) === null) {
+      setVisibility(true)
+    }
+
+    // And set window.onfocus callback after each the user return
```

Comment:
Don't understand the "after each the user return" part

Reply:
change, @alx-khramov see pls